rpc/client/helpers.go

```diff
@@ -42,7 +42,7 @@ func WaitForHeight(c StatusClient, h int, waiter Waiter) error {
 		if err != nil {
 			return err
 		}
-		delta = h - s.LatestBlockHeight
+		delta = int(h - s.LatestBlockHeight)
```

Comment:
?

Reply:
latestblockheight is int64. delta is int

Reply:
shouldnt we change delta to int64?

Reply:
yes!